css/amp.css

```diff
@@ -831,3 +831,18 @@ amp-story .i-amphtml-loader {
 [amp-fx^="fly-in"] {
   visibility: hidden;
 }
+
+/**
+ * amp-addthis
+ */
+
+/**
+ * Make amp-addthis tool floating if user give data-widget-type
+ */
+
+amp-addthis[data-widget-type="floating"] {
```

Comment:
/cc @kristoferbaxter FYI. Small CSS needed before render time to ensure non content shifting as element's layout fully change. Moving this to extension CSS causes FOUC since they are async.